package/com.unity.formats.usd/Editor/Scripts/UsdBuildPostProcess.cs

```diff
@@ -24,6 +24,13 @@ public class UsdBuildPostProcess {
 #if UNITY_EDITOR_OSX
```

Comment:
Nitpicking:
at least inside unity trunk, ifdefs are discouraged. I would make a switch/case since target tells you what platform are you building.
If not, at at the very least get rid of **ELSE**( if you build a switch/PS4 or whatever strange game, you execute the **ELSE** part)

Reply:
Thinking more about this, this is VERY WRONG :)
You can build multiple platforms from one editor. UNITY_EDITOR_* is not the correct mechanism.

Reply:
I'm fine with fixing it on this branch (and completely understand your reasoning), I was just sticking with the convention that's already in place, adding in the elif.